tiledb/cloud/client.py

```diff
@@ -13,6 +13,7 @@
 from tiledb.cloud import tiledb_cloud_error
 from tiledb.cloud._results import decoders
 from tiledb.cloud._results import results
+from tiledb.cloud.pool_manager_wrapper import PoolManagerWrapper
```

Comment:
nit: in general this code now uses the convention of "import modules, not things within modules", so `from tiledb.cloud import pool_manager_wrapper` would be the thing to do here (with the consideration of the following comment…)

Reply:
What's the benefit of the suggested change? Can you please elaborate?

Reply:
We don't use it quite everywhere here, but the idea is to make it easy to tell exactly where any symbol is coming from at its usage site. I picked it up from [the Google style guide](https://google.github.io/styleguide/pyguide.html#22-imports), where the rationale is explained:

> The namespace management convention is simple. The source of each identifier is indicated in a consistent way; `x.Obj` says that object `Obj` is defined in module `x`.